### ISSUE TITLE
Handsfree Speaker Connection

### ISSUE THREAD

**Issue by mbt28** [tedsalmon/BlueBus#71]
Hi,

How is the handsfree speaker which is located under the steering wheel works? Is CD53 unit driving it? Or bluebus itself drives it with and external connector?

I have standard speakers with cd53, I have handsfree speaker but waiting for retrofitting.

Thanks,

**Comment by piersholt** [comment]
If you could specify which model and year the vehicle is, that would be a great help. There's a multitude of telephone configurations between years, models, options etc.

**Comment by tedsalmon** [comment]
@mbt28,

I'll take a wild guess that you're talking about an early (1999 - 2001) E46, right? If so, the speaker under the steering column is directly driven by the BlueBus, and the BlueBus has an amplifier dedicated to this. The TELNF+/- signal lines route from the telephone connector to the speaker in question on these cars. The same is applicable to E38, E39, E52 and E53 models that came without DSP the amplifier, except those cars received dual cone front door speakers, one cone was driven for media audio, and the other was driven for telephony by the TCU.

In later E46, all E8x cars and DSP Amplifier equipped cars, the BlueBus pushes call audio through the same TELNF+/- line, but it is routed to the Radio or the DSP depending on chassis. From there, the respective units drive the front door speakers.

-Ted

**Comment by tedsalmon** [comment]
Closing this issue due to lack of response. I think we've given all the necessary information anyways :)

**Comment by mbt28** [comment]
Hi Ted,

Sorry for late response, due to some issues I had to left my car in garage for a long time. 

I have this head unit: Alpine, Model CD53, Part Number 65. 12-6 919 072

The car itself from 2002/01, no phone and cd changer prewiring exist. I ordered bluetooth antenna and bluetooth bracket for behind the rear seat mounting. Does my head unit have phone input or should I use the handsfree speaker? I will do the wiring from glovebox to bluetooth unit via 54pin connector.
